frontend/src/components/JiraImportServerData.svelte

```diff
@@ -0,0 +1,136 @@
+                                appearance-none rounded w-full py-2 px-3
+                                text-gray-700 leading-tight focus:outline-none
+                                focus:bg-white focus:border-purple-500"
+                                    id="userName"
+                                    required/>
+                        </div>
+                    </div>
+                    <div class="mb-4">
+                        <label
+                                class="block text-gray-700 text-sm font-bold mb-2"
+                                for="password">
+                            {$_('pages.jiraRestCfg.fields.password.label')}
+                        </label>
+                        <div class="control">
+                            <input
+                                    bind:value="{password}"
+                                    placeholder="{$_('pages.jiraRestCfg.fields.password.placeholder')}"
+                                    class="bg-gray-200 border-gray-200 border-2 appearance-none rounded
+                                w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none
+                                focus:bg-white focus:border-purple-500"
+                                    id="password"
+                                    name="password"
+                                    type="password"
+                                    required/>
+                        </div>
+                    </div>
+                    <div class="mb-4">
+                        <label
+                                class="block text-gray-700 text-sm font-bold mb-2"
+                                for="apiEndpoint">
+                            {$_('pages.jiraRestCfg.fields.apiEndpoint.label')}
+                        </label>
+                        <div class="control">
+                            <input
+                                    name="apiEndpoint"
+                                    bind:value="{apiEndpoint}"
+                                    placeholder="{$_('pages.jiraRestCfg.fields.apiEndpoint.placeholder')}"
+                                    class="bg-gray-200 border-gray-200 border-2
+                                appearance-none rounded w-full py-2 px-3
+                                text-gray-700 leading-tight focus:outline-none
+                                focus:bg-white focus:border-purple-500"
+                                    id="apiEndpoint"
+                                    required/>
+                        </div>
+                    </div>
+                    <div class="mb-4">
+                        <label
+                                class="block text-gray-700 text-sm font-bold mb-2"
+                                for="jql">
+                            {$_('pages.jiraRestCfg.fields.jql.label')}
+                        </label>
+                        <div class="control">
+                            <input
```

Comment:
I think from a UX perspective the user should be able to see the result of the JQL and select only the stories they actually want.  Also I'm not too versed in JQL so I assume most end users wont be as well, will need to consider another option such as fetching stories from a specific epic or backlog somehow.